extensions/src/main/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessor.java

```diff
@@ -731,10 +731,18 @@ private void performBackup()
     if ( !authResponse.equals( "true" ) ) {
       System.err.println( Messages.getInstance().getString( "CommandLineProcessor.ERROR_0006_NON_ADMIN_CREDENTIALS" ) );
     }
+
+    // Build the complete URL to use
+    String backupURL = contextURL + API_REPO_FILES_BACKUP;
+
+    // BISERVER-14935: In case "requestParameterAuthenticationEnabled" is set to 'true'
+    backupURL += ( "?userid=" + getUsername() + "&password=" + Encr.encryptPassword(
+      Encr.decryptPasswordOptionallyEncrypted( getPassword() ) ) );
+
     WebResource resource = client.resource( backupURL );
 
     // Response response
-    Builder builder = resource.type( MediaType.MULTIPART_FORM_DATA ).accept( MediaType.TEXT_HTML_TYPE );
+    Builder builder = resource.type( MediaType.APPLICATION_FORM_URLENCODED ).accept( MediaType.TEXT_HTML_TYPE );
```

Comment:
Why did you change from MULTIPART_FORM_DATA into APPLICATION_FORM_URLENCODED ? Usually MULTIPART_FORM_DATA is more efficient.

Reply:
Using MULTIPART_FORM_DATA results in a "no multipart boundary was found".